pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-firewall-models"
-version = "0.1.0-beta.3"
+version = "1.0.0"
```

Comment:
perhaps start with `version = "1.0.0-beta.1"`?

Reply:
This would be the 4th beta and not gotten much feedback from the community. I'm thinking beta release is holding adoption back. I'm also fairly confident in the quality of the release